src/skins/Adventure/components/Content/Contact.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {Basics} from "../../../../models";
+import {utils} from "../../utils/Utils";
+import "./Contact.scss"
+
+export function ContentComponent(props: { data: Basics, screenSize: number, scrollLeft: number, scrollTop: number }) {
+  const MOUNTS_MOVEMENT_RATIO = 0.1;
+  const MOUNTS_STARTING_OFFSET = 0.9;
+
+  return (
+
+    <div className="lvl-contact night">
+      <div>
+      </div>
+      <div className=" ground-night" >
+        <div className="wooden-ground-night" style={{backgroundPositionX: utils.moveElement(props.screenSize * MOUNTS_MOVEMENT_RATIO, props.scrollLeft, 0.2)}}>
+        <div className="wooden-ground-night-2"
+               style={{backgroundPositionX: utils.moveElement(props.screenSize * MOUNTS_STARTING_OFFSET, props.scrollLeft, 0.2)}}></div>
+        </div>
+      </div>
+      <div className="heading contact">
+        <h1>Contact</h1>
+      </div>
+      <div className="heading contact-list">
+        <div><img src={props.data.picture} height={100} width={100}  alt={props.data.name}/></div>
+        <ul>
+          <li className="skill-item">{props.data?.name}</li>
+          <li className="skill-item">{props.data?.email}</li>
+          <li className="skill-item">{props.data?.phone}</li>
+          <li className="skill-item">{props.data?.picture}</li>
+          <li className="skill-item">{props.data?.picture}</li>
+          <li className="skill-item">{props.data?.url}</li>
+        </ul>
+        {props.data.profiles &&
```

Comment:
add data.profiles > 0 , 
we'll have some strange 0 prints on the screen , I had this issue before when the array is empty

Reply:
@Nemo9439 I already have this:
![image](https://user-images.githubusercontent.com/24668564/97800943-cfd18980-1c41-11eb-8176-c4789a046777.png)